cli/Cargo.toml

```diff
@@ -15,21 +15,38 @@ categories = ["cli", "tool"]
 include = ["/Cargo.toml", "src/**/*.rs", "/README.md", "/LICENSE"]
 
 [dependencies]
+env_logger = "0.6"
 derive_more = "0.14.0"
 structopt = "0.2.15"
 itertools = "0.8"
+log = "0.4"
 heck = "0.3"
+futures = "0.1.28"
+jsonrpc-core-client = { version = "13.0", features = ["ws"] }
+toml = "0.5"
 zip = { version = "0.5", default-features = false }
 pwasm-utils = "0.11.0"
 parity-wasm = "0.40.2"
 cargo_metadata = "0.8.2"
+runtime_primitives = { git = "https://github.com/paritytech/substrate/", package = "sr-primitives" }
+serde = { version = "1.0", features = ["derive"] }
+srml-contracts = { git = "https://github.com/paritytech/substrate/", package = "srml-contracts" }
+srml-balances = { git = "https://github.com/paritytech/substrate/", package = "srml-balances" }
+srml-system = { git = "https://github.com/paritytech/substrate/", package = "srml-system" }
+srml-indices = { git = "https://github.com/paritytech/substrate/", package = "srml-indices" }
+substrate-primitives = { git = "https://github.com/paritytech/substrate/", package = "substrate-primitives" }
+subxt = { git = "https://github.com/paritytech/substrate-subxt/", package = "substrate-subxt" }
```

Comment:
Let's get rid of as many direct SRML dependencies as we can.

Reply:
I'm going to move the default `Runtime` definition to `subxt` so we can remove these direct dependencies.